config.properties

```diff
@@ -1,3 +1,10 @@
+#Driver configuration
 driver=webdriver.chrome.driver
 path=/Users/marharyta-horbunova/Documents/project/webdrivers/chromedriver
+
```

Comment:
Implementation across browser usage should be done either through config or env variable.